hack/e2e-tests.sh

```diff
@@ -108,8 +106,9 @@ fi
 test.setup
 
 #Run the tests
-if [ "$skipTests" = false ]; then
-  $SOURCE_ROOT/hack/tests/test_default_deploy.sh $@
-  $SOURCE_ROOT/hack/tests/test_standalone_docker.sh $@
-  $SOURCE_ROOT/hack/tests/test_heapster.sh $@
-fi
+[ "$skipTests" = false ] || exit 0
```

Comment:
Can you please keep this in an if statement instead of an exit? Its easier to read what is going on and means that people can easily add extra steps following this section if needed in the future.